src/platform/forms-system/test/js/validation.unit.spec.js

```diff
@@ -402,8 +402,8 @@ describe('Schemaform validations', () => {
       const errors = { to: { addError: sinon.spy() } };
       validateDateRangeAllowSameMonth(errors, {
         // the difference from validateDateRange
-        from: '2014-01-XX',
-        to: '2014-01-XX',
+        from: '2014-01-04',
+        to: '2014-01-05',
```

Comment:
I think we may need to allow for `XX`. The forms library might actually be using that as the placeholder string. If so, the tests need to use it as well.